code/src/styles/MovieList.css

```diff
@@ -0,0 +1,125 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.list-movie-wrapper {
+  display: grid;
+  grid-template-columns: repeat(5,1fr);
+  grid-auto-rows: auto;
+  grid-gap: 0;
+  grid-row-gap: 0;
+  row-gap: 0;
+  justify-items: center;
+  justify-content: center;
+  width: 100vw;
+  background: #000;
+}		
+	
+.list-movie-item {
+  position: relative;
+  max-height: 100%;
+}
+
+.list-movie-poster {
+  width: 100%;
+  height: 100%;
+}
+
+.list-details-wrapper {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1.5rem 1.5rem 3rem 1.5rem;
+  background: rgba(44, 21, 29, 0.9);
+  opacity: 0;
+  transition: .5s ease; 
+}
+
+.list-title {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: 600;
+  font-size: 36px;
+  text-align: center;
+  color: #fff;
+  margin-top: 50%;
+  opacity: 0;
+  transition: transform .5s ease; 
+}
+
+.list-release {
+  font-family: 'Montserrat', sans-serif;
+  color: #fff;
+  font-weight: 500;
+  opacity: 0;
+  transition: transform .5s ease; 
+}
+
+.list-movie-item:hover 
+.list-details-wrapper,
+.list-title,
+.list-release {
+  opacity: 1;
+}
+
+.list-movie-item:hover .list-title {
+  transform: scale(1.1);
+}
+
+.list-movie-item:hover .list-release  {
+  transform: scale(1.1);
+}
+
+
+@media(min-width:1710px) {
+  .list-movie-wrapper {
+    grid-template-columns: repeat(6,1fr);
+  }
+}
+
+@media(max-width:1368px) {
+  .list-movie-wrapper {
+    grid-template-columns: repeat(4,1fr);
+  }
+}
+
+@media(max-width:1024px) {
+  .list-movie-wrapper {
+    grid-template-columns: repeat(3,1fr);
+  }
+}
+
+@media(max-width:684px) {
+  .list-movie-wrapper {
+    grid-template-columns: repeat(2,1fr);
+  }
+
+  .nav-heading {
+    font-size: 2.5rem;
+  }
+}
+
+@media(max-width:450px) {
+  .list-movie-wrapper {
+    grid-template-columns: 1fr;
+  }
+
+  .navigation {
+    justify-content: center;
+  }
+
+  .nav-heading {
+    margin-left: 0;
+    margin-bottom: 0.7rem;
+  }
+
+  .list-movie-item {
+    width: 100vw;
+  }
+}
```

Comment:
You are using a lot of different break points, which is hard to read and understand. I think you should style mobile first and then add queries on tablet and desktop when you find it necessary.